doc/src/build/install.md

```diff
@@ -79,44 +81,53 @@ After you install Rust, proceed to the prerequisites for your operating system.
 
 ## Linux prerequisites 
 
-Install the prerequisites listed in this section. You should make sure that your system has the latest version of `apt`. Use the following command to update it:
-`sudo apt-get update`
+Take Ubuntu as an example, install the prerequisites listed in this section. You should make sure that your system has the latest version of `apt-get`. Use the following command to update it:
+
+```shell
+$ sudo apt-get update
+```
 
 ### cURL
 
 Install cURL with the following command:
-`sudo apt install curl`
+```shell
+$ sudo apt install curl
+```
 
 Verify that cURL installed correctly with the following command:
-`curl --version`
+```shell
+$ curl --version
+```
 
 ### Git CLI
 
 Run the following command to install Git, including the Git CLI:
 
-`sudo apt-get install git-all`
+```shell
+$ sudo apt-get install git-all
+```
 
-For more information, see [Install Git on Linux](https://github.com/git-guides/install-git#install-git-on-linux)
+For more information, see [Install Git on Linux](https://github.com/git-guides/install-git#install-git-on-linux).
 
 ### CMake
 
-Install CMake with the following commands:
-
-`./bootstrap`
-`make`
-`make install`
-
-For more information, see [Install CMake](https://cmake.org/install/)
+Install CMake, see [Install CMake](https://cmake.org/install/)
```

Comment:
the old commands are confusing. `./bootstrap` is the command in the [Install CMake](https://cmake.org/install/) page. Without reading that it is nowhere to find